tests/test_tutorials.py

```diff
@@ -0,0 +1,43 @@
+"""
+Discover and run tutorial scripts as unit tests.
+"""
+
+import os
+import pytest
+import subprocess
+import sys
+
+from pathlib import Path
+
+pytestmark = pytest.mark.skipif(
```

Comment:
This doesn't seem to be user later on.

Reply:
I thought that too... but if I remove it, change the name, or use '_' the line doesn't work. It turns out it is a global variable

https://docs.pytest.org/en/stable/example/markers.html#marking-whole-classes-or-modules

I'll add a note.

Reply:
Got it. That's fine, then.